eq-author-api/src/validation/schemas/entities.json

```diff
@@ -1,31 +1,47 @@
 {
   "$id": "http://example.com/schemas/entities.json",
   "definitions": {
-    "page": {
+    "section": {
       "type": "object",
       "properties": {
         "id": { "type": "string" },
         "title": {
-          "$ref": "definitions.json#/definitions/populatedString"
+          "type": "string"
+        },
+        "pages": {
+          "type": "array",
+          "items": {
+            "$ref": "entities.json#/definitions/page"
+          }
         }
       },
-      "required": ["id", "title"]
+      "required": ["id", "title", "pages"]
     },
-    "section": {
+    "page": {
       "type": "object",
       "properties": {
         "id": { "type": "string" },
         "title": {
-          "type": "string"
+          "$ref": "definitions.json#/definitions/populatedString"
         },
-        "pages": {
+        "answers": {
           "type": "array",
           "items": {
-            "$ref": "entities.json#/definitions/page"
+            "$ref": "entities.json#/definitions/answer"
           }
         }
       },
-      "required": ["id", "title", "pages"]
+      "required": ["id", "title"]
+    },
+    "answer": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "label": {
```

Comment:
I don't think label has to be populated for all answer types which this is currently matching.

Reply:
Now restricted to TextField input type. We'll need to remove this restriction once we want it to apply to all label fields